app/src/main/java/com/onegini/mobile/exampleapp/view/activity/AuthenticationActivity.java

```diff
@@ -44,8 +45,8 @@
   @SuppressWarnings({ "unused", "WeakerAccess" })
   @Bind(R.id.authenticator_message)
   TextView authenticatorMessage;
-
-  @SuppressWarnings({"unused"})
+  @Nullable
```

Comment:
I would prefer to leave it as is, this way you won't have to add cancel button if you won't use it. I think, that we included cancel  button in views, which are using it.